x-pack/platform/plugins/shared/alerting/common/routes/maintenance_window/external/apis/find/schemas/v1.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { maintenanceWindowResponseSchemaV1 } from '../../../response';
+import { maintenanceWindowStatusSchema } from '../../../../shared/schemas/v1';
+import { maxMaintenanceWindowDocs } from '../../../../shared/constants/v1';
+import { validatePagination } from '../../../../shared/validation/v1';
+
+export const findMaintenanceWindowsQuerySchema = schema.object(
+  {
+    title: schema.maybe(
+      schema.string({
+        meta: {
+          description: 'The title of the maintenance window.',
+        },
+      })
+    ),
+    created_by: schema.maybe(
+      schema.string({
+        meta: {
+          description: 'The user who created the maintenance window.',
+        },
+      })
+    ),
+    status: schema.maybe(
+      schema.oneOf([maintenanceWindowStatusSchema, schema.arrayOf(maintenanceWindowStatusSchema)], {
+        meta: {
+          description:
+            'The status of the maintenance window. One of "running", "upcoming", "finished" or "archived".',
+        },
+      })
+    ),
+    page: schema.number({
+      defaultValue: 1,
+      min: 1,
+      max: maxMaintenanceWindowDocs,
```

Comment:
Thanks for calling this schema to my attention @js-jankisalvi. I assumed it was correct because it was already in use in the internal route, but check this out. Why is the maximal page number equal to the total number of docs? Shouldn't it be `totalAmount/maxPageSize`?

Line 48 is also wrong imo. The minimum amount per page should not be 0.

I'm moving all this to shared and changing the values.

Reply:
Yup, this is also wrong 😅 

>The minimum amount per page should not be 0.

 Yeah, i think we usually set it to 10